app/src/main-process/menu/build-spell-check-menu.ts

```diff
@@ -0,0 +1,112 @@
+import { app, BrowserWindow, MenuItem } from 'electron'
+
+export async function buildSpellCheckMenu(
+  window: BrowserWindow | undefined
+): Promise<ReadonlyArray<MenuItem> | undefined> {
+  if (window === undefined) {
+    // window.webContents.off('context-menu', mergeDeferredContextMenuItems)
```

Comment:
This used to be a thing before this PR. Unless I'm missing something, it looks it's not needed anymore so we can probably remove this line.

Reply:
Ah thanks, I forgot to remove that.  I copied that over in case I would need it, but I am not sure why it was there in the first place since we are using `.once` and it seems to work correctly without it... Only scenario I thought of was if you opened another context menu very quickly (before the `context-menu` event fires), which in the original logic flow could mean merging with the wrong context menu since we were storing it. Which again means we shouldn't need this.